src/platform/user/authentication/components/LoginActions.jsx

```diff
@@ -34,6 +40,8 @@ export default function LoginActions({ externalApplication }) {
 
   return (
     <div className="row">
+      {externalApplication == null &&
```

Comment:
Change `==` to `===` (JS best practices. == will try to cast to a type if possible, whereas === will do strict type checking)

Reply:
Interesting, @asg5704 when I `console.log(externalApplication === null)` it returns false locally. Is this intended behavior, I think on `http://localhost:3001/?next=loginModal&oauth=true` externalApplication is undefined? 

Not sure if there is a difference between local vs other env's..

Reply:
Ah sorry @moanu404 I was looking strictly at the code. What you can do instead is `!externalApplication && rest of yoru code` this will check for `null` or `undefined` my apologies.

Reply:
Copy that! No worries haha, I was totally on autopilot from ruby with only using `==` . I just made the change, TIA for looking :)